### PR TITLE
[Improve] improve vue-vben-admin's license issue

### PR DESCRIPTION
**Pull request by wolfboys**
[Improve] improve vue-vben-admin's license issue

1. add the LICENSE file from https://github.com/vbenjs/vue-vben-admin/blob/main/LICENSE to [ui-licenses](https://github.com/apache/incubator-streampark/tree/dev/dist-material/release-docs/licenses/ui-licenses)
2. Remove "streampark-console/streampark-console-webapp/LICENSE", There is already a LICENSE in the root path, We don't need this LICENSE.

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @tisonkun PTAL thx.

File: streampark-console/streampark-console-webapp/LICENSE
Comment (tisonkun) [inline streampark-console/streampark-console-webapp/LICENSE:1]: Then I suggest we update the README file to clearly declare that this console is a fork of `vue-vben-admin` and add alink to the LICENSE-vue-vben-admin.txt file.

File: dist-material/release-docs/licenses/ui-licenses/license-vue-vben-admin.txt
Comment (tisonkun) [inline dist-material/release-docs/licenses/ui-licenses/license-vue-vben-admin.txt:1]: You should then correspondingly update the information at the root LICENSE, which writes:

> The text of the license is included in streampark-console/streampark-console-webapp/LICENSE-vue-vben-admin.txt

And you move the license file now.

**Comment by tisonkun** [comment]
Push a commit to resolve my comments.

**Review by tisonkun — APPROVED** [review]

**Comment by wolfboys** [comment]
> Push a commit to resolve my comments.

Thanks for the driving, it's exciting to have a professional push, it's looking a lot better now.

**Pull request merged**